react/features/filmstrip/components/native/InFocusView.js

```diff
@@ -176,6 +176,10 @@ class InFocusView extends Component<Props> {
     }
 
     _createSelfFrameVideoComponent() {
+        if (this.props.isSimplifiedConference) {
```

Comment:
Returning a null value is not too good. It will be much better, if we check the isSimplifiedConference property before we call the _createSelfFrameVideoComponent function. Like in case of _createWrapUpButtonsPlaceholder().

Reply:
Ok, I'll make the change